### DISCUSSION TITLE
Qiitaのviewsの数とGoogleアナリティクスのビュー数の相違

### DISCUSSION THREAD

**Discussion by sjuny** [increments/qiita-discussions#919] [ヘルプ]
Qiitaのviewsの定義はどのようになっていますでしょうか。また、Googleアナリティクスのビューとの差異がありますが、何故でしょうか。

**Comment by tanaka-asami415** [comment]
@sjuny 
お問合せありがとうございます。
Qiitaのview数は、Googleアナリティクスの計測の仕組みと異なるため、数値が異なる場合がございます🙇

**Comment by sjuny** [comment]
@tanaka-asami415 

ご回答ありがとうございます。
仕組みが異なることはわかりました。
